core/templatetags/mail_tags.py

```diff
@@ -0,0 +1,25 @@
+from django.template import Library, Template
+
+register = Library()
+
+
+@register.inclusion_tag("emails/_body_content.html", takes_context=True)
```

Comment:
Why are these custom tags, rather than `{% include ... with content= ... %}`?

Reply:
when using `{% include ... with content= ... %}` it would not evaluate the template variables that were being passed to the included template and would treat them as string literals (or django would error when parsing the template that it couldnt recognize the tokens )

example email with
 ` {% include "emails/_body_content.html" with content="A password reset was requested for your account ({{reset.user.email}}) at {{Config.system.site_name}} (https://{{settings.MAIN_DOMAIN}})." %}`

![image](https://user-images.githubusercontent.com/29409178/210116335-756cb567-f97d-4257-bb8c-e12c123786fe.png)

Reply:
Ah yes, fair enough, and this is the reason `inclusion_tag` even exists I believe! (well, for the admin, but similar issue)